docs/java-rest/high-level/supported-apis.asciidoc

```diff
@@ -564,12 +564,14 @@ include::ilm/lifecycle_management_status.asciidoc[]
 include::ilm/retry_lifecycle_policy.asciidoc[]
 include::ilm/remove_lifecycle_policy_from_index.asciidoc[]
 
-== Data Frame APIs
+[[_data_frame_transform_apis]]
```

Comment:
Is this anchor supposed to start with an underscore?

Reply:
Yes, that's what it defaults to automatically at the moment.  I think we'll want to ultimately change it, but didn't want to break any current bookmarks.